electrum_nmc/electrum/tests/test_bitcoin.py

```diff
@@ -181,9 +181,11 @@ def sign_message_with_wif_privkey(wif_privkey, msg):
         sig1_b64 = base64.b64encode(sig1)
         sig2_b64 = base64.b64encode(sig2)
 
-        # Re-signed these with Namecoin Core, since the upstream Electrum ones are invalid for Namecoin's msg_magic.
-        self.assertEqual(sig1_b64, b'IKqxjcFykcFJPUsIJtUvR5901nJnD/WN326bDVHqnvxjX6+E/mXH9FY+MNpNyl/liXQDhd53BihaVH2lOGknzFU=')
-        self.assertEqual(sig2_b64, b'HMBdVzZJfSqhGsrJ6NgUoLUxPmTS0NSxA2Y/q3Te69MVNtt4aWbJORq+0MCllDfqiKo9IIaWpSmXk0VXlaIMxx4=')
+        # Due to a difference in low-s signing, Namecoin Core and Electrum
+        # produce different signatures.  The values here are produced by
+        # Electrum but have been checked to verify with Namecoin Core.
```

Comment:
The constraint added is Low-R, not Low-S.  `s/Electrum/Electrum-NMC/` for consistency with the mention of Namecoin Core.  Also please link to the Electrum PR that introduced this change ( https://github.com/spesmilo/electrum/pull/5820 ) so that someone running across this code knows where to find further context.

Reply:
Good points, I've fixed the comment accordingly.